osal/src/os/posix/src/os-impl-dirs.c

```diff
@@ -32,11 +31,10 @@
 #include <fcntl.h>
 #include <dirent.h>
 #include <sys/stat.h>
-#include <sys/types.h>
-#include <sys/wait.h>
 
-#include "os-posix.h"
 #include "os-impl-dirs.h"
+
+#include "os-freertos.h"
```

Comment:
There were other locations where `os-posix.h` was removed but `os-freertos.h` was not added in those instances, should we add the freeRTOS include to those locations as well?